File: examples-sdk/swift/018-basic-ai-multi-model-router/main.swift
// 018-basic-ai-multi-model-router — Swift SDK equivalent
// Compile: vil compile --from swift --input 018-basic-ai-multi-model-router/main.swift --release

let server = VilServer(name: "ai-multi-model-router", port: 3085)
let router = ServiceProcess(name: "router")
router.endpoint(method: "POST", path: "/route", handler: "route_handler")
server.service(router)
server.compile()

File: examples-sdk/kotlin/018-basic-ai-multi-model-router/main.kt
// 018-basic-ai-multi-model-router — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 018-basic-ai-multi-model-router/main.kt --release

fun main() {
    val server = VilServer("ai-multi-model-router", 3085)
    val router = ServiceProcess("router")
    router.endpoint("POST", "/route", "route_handler")
    server.service(router)
    server.compile()
}
